backend/games/battle_mogs/src/transitions/hatch.rs

```diff
@@ -87,15 +88,7 @@ where
 		])
 	}
 
-	fn segment_and_bake(mogwai: &mut Mogwai, hash: &[u8; 32]) -> ([[u8; 32]; 2], RarityType) {
-		let block_hash = unsafe {
-			let mut block_hash: MaybeUninit<[u8; 32]> = MaybeUninit::uninit();
-			let block_hash_ptr = block_hash.as_mut_ptr() as *mut u8;
-			copy_nonoverlapping(hash.as_ref()[0..32].as_ptr(), block_hash_ptr, 32);
-			block_hash.assume_init()
-		};
-
```

Comment:
To me, it seems that this entire block was a no-op. So I removed this piece of code. Can you confirm this?

Reply:
Yeah, Its probably a leftover